### DISCUSSION TITLE
Question regarding proxy_cobalt_malformed_uas

### DISCUSSION THREAD

**Discussion by orionova** [SigmaHQ/sigma#3126] [Q&A] [answered]
Hey, I'm trying to fully understand the following rule's logic:
[proxy_cobalt_malformed_uas.yml](https://github.com/SigmaHQ/sigma/blob/master/rules/proxy/proxy_cobalt_malformed_uas.yml)

The rule lists 3 UAs (IE 7,10) and a UA suffix. Usage of Cobalt Strike doesn't require using these necessarily [reference](https://posts.specterops.io/a-deep-dive-into-cobalt-strike-malleable-c2-6660e33b0e0b).

What led to choosing the used logic in this rule ?

**Comment by nasbench** [comment, marked as answer]
Hey @orionova 

You're correct, CS doesn't require the usage of these UA as you can use any. The rule is using a real life example of profiles from the GH repo linked as a reference in the rule (at the time of the writing).

Hence they are suspicious and could catch a subset of what's possible.
